contracts/truefi/TrueLender.sol

```diff
@@ -111,9 +112,32 @@ contract TrueLender is Ownable {
         pool.borrow(amount);
         currencyToken.approve(address(loanToken), amount);
         loanToken.fund();
+        loans.push(loanToken);
         emit Funded(address(loanToken), amount);
     }
 
+    function valueFor(ILoanToken loanToken) public view returns (uint256) {
+        (uint256 amount, uint256 apy, uint256 duration) = loanToken.getParameters();
+        uint256 start = loanToken.start();
+        uint256 passed = block.timestamp.sub(start);
+        if (passed > duration) {
+            passed = duration;
+        }
+
+        uint256 helper = amount.mul(apy).mul(passed).mul(loanToken.balanceOf(address(this)));
+        uint256 interest = helper.div(360 days).div(10000).div(loanToken.totalSupply());
+
+        return amount.add(interest);
+    }
+
+    function value() external view returns (uint256) {
+        uint256 totalValue;
+        for (uint256 index = 0; index < loans.length; index++) {
+            totalValue = totalValue.add(valueFor(loans[index]));
+        }
+        return totalValue;
+    }
+
     function reclaim(ILoanToken loanToken) external onlyOwner {
```

Comment:
I think we should remove loans from array here as well

Reply:
Let's add this in next PR.